terraform/aks/03-aks00/20-aks.tf

```
@@ -49,4 +49,17 @@ module "kubernetes" {
   kubernetes_cluster_version         = var.kubernetes_cluster_version
 
   tags = local.common_tags
+
+  additional_node_pools = [
+    {
+      name                = "msnode"
+      vm_size             = "Standard_DS3_v2"
```

could probably re-use the other var for this?

```suggestion
      vm_size             = var.kubernetes_cluster_agent_vm_size
```